### PR TITLE
Mock data for inventory

### PR DESCRIPTION
**Pull request by irfankhaan**
Created a mock json file for inventory data

### CODE REVIEW

**Comment by irfankhaan** [comment]
@iamareebjamal, please review my PR.

**Comment by iamareebjamal** [comment]
Why am I seeing team JSON?

**Comment by irfankhaan** [comment]
I pushed the inventory.json.

**Comment by iamareebjamal** [comment]
This link should have only 1 file https://github.com/open-roboclub/roboclub-vue/pull/306/files

@Haider8 Can you please help him?

**Comment by irfankhaan** [comment]
Deleted the team json, @iamareebjamal.

File: inventory.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+"inventory" : {
+	"micro-controllers" : {
```
Comment (iamareebjamal) [inline inventory.json]: Implement as a list, not a dictionary

File: inventory.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		"arduino" : {
+			"name" : "Arduino Uno",
+			"instore" : 4,
+			"status" : "Available"
```
Comment (iamareebjamal) [inline inventory.json]: status is not needed

File: inventory.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
+"inventory" : {
+	"micro-controllers" : {
+		"arduino" : {
+			"name" : "Arduino Uno",
+			"instore" : 4,
```
Comment (iamareebjamal) [inline inventory.json]: Separate in total, working, alotted
available count will be computed

**Comment by iamareebjamal** [comment]
Good work

**Comment by iamareebjamal** [comment]
Also, learn to work with branches. Don't ever use master to send PR. Always use a separate branch for each PR

**Comment by irfankhaan** [comment]
Made the specified changes to the inventory.json. Please review, @iamareebjamal!

File: inventory.json [hunk 4]
```diff
@@ -1,53 +1,63 @@
-"inventory" : {
-	"micro-controllers" : {
```
Comment (iamareebjamal) [inline inventory.json]: Why did you remove this?

File: inventory.json [hunk 5]
```diff
@@ -1,53 +1,63 @@
...
-		"jumper" : {
-			"name" : "Jumper Cables",
-			"instore" : 100
-		},
-		"copperwire" : {
-			"name" : "Copper Wires",
-			"instore" : 50
-		}
-	}
+"arduino" : {
```
Comment (iamareebjamal) [inline inventory.json]: As I said previously, this should be a list, not dictionary

File: inventory.json [hunk 6]
```diff
@@ -1,53 +1,63 @@
...
+	"name" : "Arduino Uno",
+	"total" : 5,
+	"working" : 5,
+	"allotted" : 3,
+	"available" : 2
```
Comment (iamareebjamal) [inline inventory.json]: No need to add available. It will be calculated

File: inventory.json (hunk 5 above)
Comment (irfankhaan) [reply inventory.json]: How should I do that?

Comment (iamareebjamal) [reply inventory.json]: http://bfy.tw/OT0v

File: inventory.json [hunk 7]
```diff
@@ -1,63 +1,63 @@
...
-},
-"copperwire" : {
-	"name" : "Copper Wires",
-	"total" : 15,
-	"working" : 15,
-	"allotted" : 10,
-	"available" : 5
+"inventory" : {	
+	"microcontroller" : [
+		"arduino" : {
```
Comment (iamareebjamal) [inline inventory.json]: See the red background? That tells you that the syntax is wrong. Run through a JSON validator before submitting

**Comment by irfankhaan** [comment]
@iamareebjamal, please review my PR!

**Comment by iamareebjamal** [comment]
It's fine. I've added it to firebase. Add a security rule with read only permissions at path /inventory like downloads and robocon https://github.com/open-roboclub/roboclub-vue/blob/master/rules.bolt
